### PR TITLE
Update setupPublicCurl.sh

### PR DESCRIPTION
**Pull request by hviejo**
Add the fqdn to /etc/hosts file becouse if you only add the hostname then it is posible to lose the fqdn

### CODE REVIEW

File: scriptbuilder/src/main/resources/functions/setupPublicCurl.sh [hunk 1]
```diff
@@ -52,7 +52,7 @@ function ensure_hostname_in_hosts() {
     local ipaddr=`hostname -i`
   }
   # NOTE: we blindly trust existing hostname settings in /etc/hosts
-  egrep -q `hostname` /etc/hosts || echo "$ipaddr `hostname`" >> /etc/hosts
+  egrep -q `hostname` /etc/hosts || echo "$ipaddr `hostname -f` `hostname`" >> /etc/hosts
```
Comment (nacx) [inline scriptbuilder/src/main/resources/functions/setupPublicCurl.sh]: Should we better be explicit and use also the `-s` flag? Something like:
```bash
egrep -q `hostname -s` /etc/hosts || echo "$ipaddr `hostname -f` `hostname -s`" >> /etc/hosts
````

**Comment by nacx** [comment]
Thanks @hviejo! Just one comment. Also, could you have a look at the test failures and fix them?
https://jclouds.ci.cloudbees.com/job/jclouds-pull-requests/1618/

**Comment by hviejo** [comment]
Fixed all tests

**Comment by nacx** [comment]
Thanks! WDYT about my comment of explicitly using the `-s` parameter too? In some operating systems the `-f` might be the default behavior.

**Comment by hviejo** [comment]
Added "-s" option.
Now the /etc/hosts line is "ipAddress hostname -f hostname -s"

**Comment by nacx** [comment]
Perhaps the `-s` parameter should also be used in the "grep" part? To make sure we look for the "smaller" hostname?

**Comment by hviejo** [comment]
Yes, yes, the option are on both sides

**Comment by nacx** [comment]
>Yes, yes, the option are on both sides

Oh, I didn't refresh the page and didn't see the last commit :) Cool! If you don't mind squashing the commits into a single one I'll merge the PR.

**Comment by hviejo** [comment]
Ok, now it is squashed

**Comment by nacx** [comment]
Pushed to master as [b54c788e](https://git1-us-west.apache.org/repos/asf?p=jclouds.git;a=commit;h=b54c788e). I've fixed a few more test files in the `compute` project that failed the build locally and amended the commit. I don't know why the pull request builder has not detected that, though.
Thanks @hviejo!

**Pull request closed without merging**